packages/components/select/src/option.tsx

```diff
@@ -58,11 +48,6 @@ const OptionIcon = forwardRef<OptionIconProps, "span">(
     const { styles } = useSelectContext()
 
     const css: CSSUIObject = {
-      flexShrink: 0,
-      display: "inline-flex",
-      justifyContent: "center",
-      alignItems: "center",
-      fontSize: "0.85em",
       ...styles.itemIcon,
     }
```

Comment:
Move it to a `select` theme instead of `multi-select`.

Reply:
Sorry, I fixed it.
6b24f6702ee04ae59d09c58e3f31004515c6056e